docs/Lookup-batchjoiner.md

```diff
@@ -16,10 +16,8 @@ This will help to provide complete information that needs to be loaded into the
 Properties
 ----------
 **Lookup dataset:** Amongst the inputs connected to the lookup transformation, this determines the input that should be
-used as the lookup dataset. This dataset will be loaded into memory and will be broadcast to all executors, 
-so it should be smaller of the inputs.Amongst the inputs connected to the lookup transformation, 
-this determines the input that should be used as the lookup dataset. This dataset will be loaded into memory 
-and will be broadcast to all executors, so it should be smaller of the inputs.
+used as the lookup dataset. Amongst the inputs connected to the lookup transformation, this determines the input that 
+should be used as the lookup dataset.
```

Comment:
This sentence is duplicated: "Amongst the inputs connected to the lookup transformation, this determines the input that should be used as the lookup dataset."